install_config/aggregate_logging.adoc

```diff
@@ -379,7 +379,11 @@ xref:../install/configuring_inventory_file.adoc#configuring-node-host-labels[nod
 starting up, using this may cause a delay in ES receiving current log records.
 
 |`openshift_logging_fluentd_hosts`
-|List of nodes that should be labeled for Fluentd to be deployed.
+|List of nodes that should be labeled for Fluentd to be deployed. The default is
+to label all nodes, for example ['--all']. The null value is
+`openshift_logging_fluentd_hosts={}`.
+To spin up Fluentd pods update the daemonset's nodeSelector to a valid label. For
```

Comment:
I think nodeSelector would need backticks is said like this. Otherwise: s/nodeSelector/node selector